app/assets/stylesheets/welcome.scss

```diff
@@ -1,158 +0,0 @@
-// Place all the styles related to the welcome controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
-@import "bootstrap-sprockets";
```

Comment:
you don't have to import this here, you should have all the imports in the ```application.scss``` file